src/components/StationsList.test.js

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import StationsList from './StationsList';
+const fn = jest.fn();
+
+test('if component renders a valid HTML element', () => {
+  const radioStationsArray = [
+    {
+      id: 123,
+      name: 'Cat Country',
+      link: 'http://17573.live.streamtheworld.com/WCTKFMAAC.aac',
+    },
+  ];
+  render(<StationsList stations={radioStationsArray} handleSelect={fn} />);
+  expect(screen.getByTestId('stationsList')).toBeInTheDocument();
+});
+
+test('if component throw error when array is empty', () => {
+  const array = [];
+  console.error = jest.fn();
+  expect(() =>
+    render(<StationsList stations={array} handleSelect={fn}></StationsList>)
+  ).toThrow();
+});
```

Comment:
As mentioned in the other CR, we don't want the component to throw, instead we want it to return an empty `<ul>`

Reply:
done :)